packages/quick_actions/android/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="io.flutter.plugins.quickactions">
+    xmlns:tools="http://schemas.android.com/tools"
+    package="io.flutter.plugins.quickactions">
 
-  <application>
-    <activity android:name=".QuickActionsPlugin$ShortcutHandlerActivity" android:exported="false"/>
-  </application>
+    <application>
+        <activity
+            android:name=".QuickActionsPlugin$ShortcutHandlerActivity"
+            android:exported="false"
+            android:relinquishTaskIdentity="true"
+            tools:targetApi="lollipop" />
```

Comment:
We need `tools:targetApi` to hide a warning in android studio regarding `android:relinquishTaskIdentity` — the previous one. We can use `targetApi — lollipop` as the plugin anyways works on devices from API at least 25. Also, in case, if the app is killed and we open it using the shortcut, it will work fine but only the first time. If you click the Home button after that and use the same shortcut again it will not work the second and following times without `android:relinquishTaskIdentity`.

Reply:
Have you tried `android:noHistory="true"` ? As `android:relinquishTaskIdentity="true"` works only on API +25, all users in APIs below that will not have the same behavior.

Reply:
Yes, exactly the same API version Android shortcuts and it's classes were added ;)

Reply:
Ahh shortcuts are API 25+. So there is no problem.

Reply:
I will wait the merge to make another PR. I found that some more changes are needed. Sorry for the trouble.